lib/robots/dor_repo/dissemination/cleanup.rb

```diff
@@ -3,15 +3,16 @@
 module Robots
   module DorRepo
     module Dissemination
-      # NOTE: This step has been merged into the end accession robot. It should no longer be used, but is left in case
-      # there are existing workflow steps that require it.
+      # This step has been merged into the end accession robot. It should no longer be used.
       class Cleanup < LyberCore::Robot
         def initialize
           super('disseminationWF', 'cleanup')
         end
 
         def perform_work
-          object_client.workspace.cleanup(workflow: 'disseminationWF', lane_id:)
+          Honeybadger.notify('[WARNING] DisseminationWF:cleanup robot is deprecated and should not be used.', context: { druid: })
+          # Oct 3 2024: just warn, do not actually do the work.  If we never see these warnings, we can get rid of this robot.
+          # object_client.workspace.cleanup(workflow: 'disseminationWF', lane_id:)
           LyberCore::ReturnState.new(status: :noop, note: 'Initiated cleanup API call.')
```

Comment:
this was only place i could find that called the "cleanup" on DSA client (which is going away in https://github.com/sul-dlss/dor-services-client/pull/472)

it shouldn't be called anymore anyway - this robot and workflow has been deprecated for a while now -- so let's just warn HB